exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,27 @@
+final class Proverb {
+    private String[] words;
+    private String initWord, qualifier, result = "";
```

Comment:
These should all be local variables (apart from `words`) in the `getProverb` method since they're only used in that method :)